### PR TITLE
Proto update

### PR DESCRIPTION
**Pull request by kkuzmin**
### Solution Description
Adding new optional `application_id` field into `collected_message` proto.

### CODE REVIEW

File: proto/common_proto.piqi_pb.js [hunk 1]
```diff
@@ -1038,6 +1047,8 @@ $root.common_proto = (function() {
                 writer.uint32(/* id 8, wireType 2 =*/66).string(message.messageTypeId);
             if (message.messageTsUs != null && message.hasOwnProperty("messageTsUs"))
                 writer.uint32(/* id 9, wireType 5 =*/77).fixed32(message.messageTsUs);
+            if (message.applicationId != null && message.hasOwnProperty("applicationId"))
```
Comment (daniel-greening) [inline proto/common_proto.piqi_pb.js:1050]: Should this be message.applicaitonId != null | "" ?

**Review by JDragovichAlertLogic — APPROVED** [review]
LGTM

File: proto/common_proto.piqi_pb.js (hunk 1 above)
Comment (kkuzmin) [reply proto/common_proto.piqi_pb.js:1050]: this is an autogenerated code by `protobufjs`

Comment (daniel-greening) [reply proto/common_proto.piqi_pb.js:1050]: OIC. I'll trust in protobuf then 👍

**Review by daniel-greening — APPROVED** [review]

**Pull request merged**